pandasdmx/message.py

```diff
@@ -92,6 +93,26 @@ class Config:
     #: that returned the Message. This is not part of the SDMX standard.
     response: Response = None
 
+    def to_pandas(self, *args, **kwargs):
+        '''
+        Convert a :inst:`Message` to :mod:`pandas` object(s) by
+        calling :func:`pandasdmx.writer.write`. 
+        The :inst:`Message` is passed implicitly.  
+        Otherwise  see the docs for :func:`pandasdmx.writer.write`.
+        
+        For  backwards compat with previous releases :class:`Message` has an identical  alias
+        :meth:`Message.write`. 
+        It is deprecated and may be removed in future versions.
+        '''
+        from pandasdmx.writer import write
+        return write(self, *args, **kwargs)
+    
+    def write(self, *args, **kwargs):
+        'Deprecated alias for :meth:`Message.to_pandas`'
```

Comment:
Simply call it "Alias for…", then use a [deprecated ReST directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated).